contrib/init/openrc/docker.confd

```diff
@@ -7,7 +7,7 @@
 #DOCKER_PIDFILE="/run/docker.pid"
 
 # where the docker daemon itself is run from
-#DOCKER_BINARY="/usr/bin/docker"
+DOCKER_BINARY="/usr/bin/dockerd"
```

Comment:
Sorry, I meant we should rename this to be `DOCKERD_BINARY` instead, so that if someone has an existing file, their `DOCKER_BINARY` setting won't be used as if it were `dockerd` to poor effect.